data/txHandler.go

```diff
@@ -107,6 +117,22 @@ func MakeTxHandler(txPool *pools.TransactionPool, ledger *Ledger, net network.Go
 		postVerificationQueue: make(chan *txBacklogMsg, txBacklogSize),
 		net:                   net,
 	}
+	handler.ctx, handler.ctxCancel = context.WithCancel(context.Background())
```

Comment:
Why is this here?

Reply:
artifact of a previous change, will be removed.